lrlex/src/lib/ctbuilder.rs

```diff
@@ -547,24 +547,31 @@ where
                     yacc_header.mark_used(&"test_files".to_string());
                     let test_glob = yacc_header.get("test_files");
                     match test_glob {
-                        Some(HeaderValue(_, Value::Setting(Setting::String(test_files, _)))) => {
-                            let path_joined = grm_path.parent().unwrap().join(test_files);
-                            for path in
-                                glob(&path_joined.to_string_lossy()).map_err(|e| e.to_string())?
-                            {
-                                let path = path?;
-                                if let Some(ext) = path.extension() {
-                                    if let Some(ext) = ext.to_str() {
-                                        if ext.starts_with("grm") {
-                                            Err(ErrorString("test_files extensions beginning with `grm` are reserved.".into()))?
+                        Some(HeaderValue(_, Value::Setting(Setting::Array(test_globs, _, _)))) => {
```

Comment:
There should/needs to be the same change to this one in nimbleparse.
We should also probably add nimbleparse invocations without inputs to `.buildbot.sh`
so the testsuite could catch it.

Reply:
Should be fixed in [c22415f](https://github.com/softdevteam/grmtools/pull/604/commits/c22415f6a805e067bc985a76cf95ef6ecf48c41c)